metadata/stream/rationale.yaml

```diff
@@ -33,6 +42,15 @@ architectures:
               signature: https://artifacts.example.com/aeng0xah6vaaVosh.vdi.xz.sig
               sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
               uncompressed-sha256: 38acb15d02d5ac0f2a2789602e9df950c380d2799b4bdb59394e4eeabdd3a662
+      azurestack:
+        release: 30.1.2.3
+        formats:
+          "vdi.xz":
+            disk:
+              location: https://artifacts.example.com/aeng0xah6vaaVosh.vdi.xz
+              signature: https://artifacts.example.com/aeng0xah6vaaVosh.vdi.xz.sig
```

Comment:
Azurestack artifacts are VHD, right?  https://github.com/coreos/fedora-coreos-tracker/pull/1213/files#diff-89470a2f5cfdc5c042424276d90f535ac06ae00e9d832f0ac4a623e2d4d011a8R51-R55

If so, the same should apply for Azure above this, too

Reply:
yes, it is VHD. it was mentioned 'vdi' earlier. I wasnt sure why.